machAeroTutorials/opt_overview.rst

```diff
@@ -12,6 +12,14 @@ There are many problems with this approach, including prohibitive computational
 Instead, we use a combination of analytic gradients, automatic differentiation, and the complex-step method to compute accurate gradients for our optimizations.
 We have spent a great deal of effort to make optimization a fairly straightforward and seamless process for the end user, so we hope you enjoy learning to use our tools!
 
+Overview of framework
+---------------------
+An optimizer is used via `pyOptSparse` to select the design variables for a cannidate design. 
+`pyGeo` warps the surface mesh according to the geometric design variables. 
+To create a volume mesh, `idwarp` propagates the deformation of the surface mesh into the volume mesh.
```

Comment:
Would just remove `To create a volume mesh`, sounds wrong and the rest of the sentence is clear to me